packages/core/src/modules/connections/messages/ConnectionInvitationMessage.ts

```diff
@@ -91,8 +91,19 @@ export class ConnectionInvitationMessage extends AgentMessage {
    * @param domain domain name to use for invitation url
    * @returns invitation url with base64 encoded invitation
    */
-  public toUrl(domain = 'https://example.com/ssi') {
+  public toUrl({
+    domain = 'https://example.com/ssi',
+    useLegacyDidSovPrefix = false,
+  }: {
+    domain: string
```

Comment:
Not a 100% sure, but with a default parameter value, should it not be `domain?: string`?

Reply:
Yes. My intention was to remove the default parameter as 'https://example.com/ssi' seems weird. But obviously forgot to remove it....

Reply:
Updated